PTGLdynamics/PTGLdynamics.py

```diff
@@ -181,7 +185,13 @@ def execute_getAttribute(elem, plotting_dir, args, filepath, output_dir, working
                        '--headerfile',
                        metavar = 'headerfile',
                        default = '',
-                       help = 'to integrate a header in your files containing 3D structural data specify the path of your header file.')
+                       help = 'to integrate a header in your mmCif files specify the path of your header file.')
+                       
+cl_parser.add_argument('-C',
+                       '--compoundfile',
+                       metavar = 'compoundfile',
+                       default = '',
+                       help = 'to integrate a header in your pdb files containing 3D structural data specify the path of your header file.')                       
```

Comment:
Did we not discuss combining both options into one? Or is there a case where the user might need both options in one run?